### DISCUSSION TITLE
T-Pot behind of proxy

### DISCUSSION THREAD

**Discussion by Adi-18** [telekom-security/tpotce#1340] [Q&A]
I have T-Pot behind of a WAF. 
My problem: when I see attacks i.e. in Tanner, I only see proxy ip. Now there should implement X-Forwarded-For option, but i still see my proxy ip
![image](https://github.com/telekom-security/tpotce/assets/29123730/6fc91622-6fa7-4eb4-9fc5-b057b2908d7e)

I cannot find any description how to config/activate X-Forwarded-For

**Comment by t3chn0m4g3** [comment]
This is beyond the scope of T-Pot. If Snare / Tanner do not support XFF headers, please open an issue on their GitHub repository @ https://github.com/mushorg.
